src/StripeTests/StripeTests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net5.0;net6.0;netcoreapp3.1;netcoreapp2.1;net461</TargetFrameworks>
```

Comment:
We're dropping netcoreapp2.1 - here and below - is that expected?

Reply:
Added it back here. Below is fine. We don't need a separate package list for every target framework.